en/releases/core_components.md

```diff
@@ -0,0 +1,55 @@
+# PX4 Core Components
+
+These core components are *guaranteed* to work with a stable point release of PX4. They form the foundation of the PX4-Autopilot and it's core functionality.
+
+## Core component criteria
+
+Each core component must have:
+- Maintainer
+- Tester
+
+## PX4 Core Components List
+
+| Core component (category) | Sub component | Maintainer | Tester |
+|---|---|---|---|
+| uORB (Architecture) |  |  |  |
+| Parameters (Architecture) |  |  |  |
+| Realtime Clock (Architecture) |  |  |  |
+| Work queue (Architecture) |  |  |  |
```

Comment:
@dagar @bkueng Anything important that is missing from this list in terms of core architecture that we officially support / maintain?

Reply:
Hard to know where the line is. There's quite a bit more for things like I2C and SPI driver support.